### PR TITLE
Relate user to posts

### PR DESCRIPTION
**Pull request by heimrih**
Users can have own posts

### CODE REVIEW

**Comment by heimrih** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -16,6 +15,7 @@
               <% end %>
                 <li><%= link_to t(".profile_header"), current_user %></li>
                 <li><%= link_to t(".setting_header"), edit_user_path(current_user) %></li>
+                <li><%= link_to t("posts.header"), new_post_path %></li>
```
Comment (chan227) [inline app/views/layouts/_header.html.erb]: I think it is better to put tab for user can direct create post rather put it into the dropdown bar, if it is a route for user to create a new post.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -57,4 +60,20 @@ def load_post
     flash[:warning] = t "posts.notfound"
     redirect_to root_path
   end
+
+  def logged_in_user
+    unless logged_in?
+      store_location
+      flash[:danger] = t "users.login_remind"
+      redirect_to login_path
+    end
```
Comment (chungpv-1008) [inline app/controllers/posts_controller.rb]:
```suggestion
    return if logged_in?
    
    store_location
    flash[:danger] = t "users.login_remind"
    redirect_to login_path
```

**Comment by heimrih** [comment]
ready

File: Gemfile [hunk 3]
```diff
@@ -22,7 +22,6 @@ gem "rails-i18n"
 gem "sass-rails", ">= 6"
 gem "turbolinks", "~> 5"
 gem "webpacker", "~> 4.0"
-gem "kaminari"
```
Comment (ngocvt-0484) [inline Gemfile]: why you delete this gem ?

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -13,10 +15,11 @@ def new
 
   def create
     @post = Post.new post_params
+    @post.user = current_user
```
Comment (ngocvt-0484) [inline app/controllers/posts_controller.rb]: @post = current_user.posts.build post_params
so remove line 15.18

File: app/views/posts/new.html.erb [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
-  <%= f.label :body %>
-  <%= f.text_area :body, class: "form-control" %>
-  <%= f.submit t("posts.submit"), class: "btn btn-primary" %>
-<% end %>
+    <%= f.label :title %>
+    <%= f.text_field :title, class: "form-control" %>
+    <%= f.label :body %>
+    <%= f.text_area :body, class: "form-control" %>
+    <%= f.submit t("posts.submit"), class: "btn btn-primary" %>
+  <% end %>
```
Comment (ngocvt-0484) [inline app/views/posts/new.html.erb:10]: wrong indent

File: Gemfile (hunk 3 above)
Comment (heimrih) [reply Gemfile]: repeated

File: app/views/posts/new.html.erb (hunk 5 above)
Comment (heimrih) [reply app/views/posts/new.html.erb:10]: There should have indent between div right?

**Comment by heimrih** [comment]
ready

**Pull request merged**